tests/wamr-test-suites/spec-test-script/runtest.py

```diff
@@ -1158,6 +1158,9 @@ def run_wasm_with_repl(wasm_tempfile, aot_tempfile, opts, r):
     tmpfile = aot_tempfile if test_aot else wasm_tempfile
     log("Starting interpreter for module '%s'" % tmpfile)
 
+    if opts.qemu:
+        tmpfile = f"/tmp/{os.path.basename(tmpfile)}"
```

Comment:
Is better to move L1159 to be after these two lines, or the log may be not so accurate?

Reply:
OK, thanks.